tests/test_scattering_analytical.py

```diff
@@ -0,0 +1,57 @@
+import pytest
+import numpy as np
+
+import imkar as ik
+
+
+def test_rectangular_under_30_degree():
+    # Test compares the directional scattering coefficient under 30° with
+    # the one shown in Figure 2 in Embrechts et al.
+    c_over_L = 343.901
+    x = np.linspace(4, 8, num=5)
+    result = ik.scattering.analytical.rectangular(
+        x*c_over_L, np.array([30]), 0.50, 1, 0.49, 343.901)
+    actual = np.reshape(np.array(
```

Comment:
here some more tests are missing, since its not covering all valid input combinations.
e.g. ik.scattering.analytical.rectangular([200, 300], [30], 0.50, 1, 0.49, 343.901)
ik.scattering.analytical.rectangular(200, 30, 0.50, 1, 0.49, 343.901)
ik.scattering.analytical.rectangular([200, 300], [20, 30], 0.50, 1, 0.49, 343.901)
sould also work
this things should be tested in a separate test